Source/Blazorise/Utilities/BaseDateTimeFormatConverter.cs

```diff
@@ -0,0 +1,85 @@
+﻿#region Using directives
+using System.Collections.Generic;
+using System.Text;
+#endregion
+
+namespace Blazorise.Utilities;
+
+/// <summary>
+/// Base class for internal date time formaters.
+/// </summary>
+public abstract class BaseDateTimeFormatConverter
+{
+    #region Methods
+
+    /// <inheritdoc/>
+    public string Convert( string format )
+    {
+        if ( string.IsNullOrEmpty( format ) || format.Length == 0 )
```

Comment:
Is this not redundant? Empty is the same as **Length 0**, no?

Reply:
Indeed.